steward/src/cellars.rs

```diff
@@ -16,30 +16,30 @@ pub const BLOCKED_ADAPTORS: [&str; 3] = [
     // Original uniswap v3 adaptor
     "7c4262f83e6775d6ff6fe8d9ab268611ed9d13ee",
     // Original vesting simple adaptor
-    "1eAA1a100a460f46A2032f0402Bc01FE89FaAB60",
+    "1eaa1a100a460f46a2032f0402bc01fe89faab60",
     // Original compound C token adaptor
-    "26DbA82495f6189DDe7648Ae88bEAd46C402F078",
+    "26dba82495f6189dde7648ae88bead46c402f078",
 ];
 pub const BLOCKED_POSITIONS: [u32; 9] = [4, 5, 6, 7, 8, 9, 10, 11, 12];
 pub const ALLOWED_SETUP_ADAPTORS: [&str; 9] = [
-    // UniswapV3Adaptor V2
+    // Uniswapv3Adaptor v2
```

Comment:
This address is unchanged -- was there a new UniswapV3 update to make it a V2 beyond the bugfix that went into the replacement last time? I believe the new one IIRC from this PR has some functions removed or changed, and if so, it's probably not the same address as what was originally here.